decidim-elections/app/permissions/decidim/elections/admin/permissions.rb

```diff
@@ -62,12 +60,12 @@ def trustee_participatory_space
           @trustee_participatory_space ||= context.fetch(:trustee_participatory_space, nil)
         end
 
-        def allow_if_not_started
-          toggle_allow(election && !election.started?)
+        def allow_if_not_blocked
+          toggle_allow(election && !election.blocked?)
         end
 
-        def allow_if_valid_and_not_started
-          toggle_allow(election && !election.started? && election.valid_questions?)
+        def allow_if_valid_and_not_blocked
+          toggle_allow(election && !election.blocked? && election.valid_questions?)
```

Comment:
@agustibr lately I've noticed that we're placing too many business logic in the permissions system. This logic should belong to the Commands, as they may be called from anywhere after a controller action and we'll loose many of the checks we're doing in the permissions system.
No need to change your code now but we should avoid placing more business logic into the permissions system please.
I'll open a discussion regarding this.

Reply:
The discussion: https://github.com/decidim/decidim/issues/6855